src/lib/writer.c

```diff
@@ -1017,21 +1017,23 @@ encrypt_se_ip_writer(const uint8_t *src,
                      pgp_writer_t * writer)
 {
     const unsigned   bufsz = 128;
+    rnp_ctx_t *      ctx;
     encrypt_se_ip_t *se_ip = pgp_writer_get_arg(writer);
     pgp_output_t *   litoutput;
     pgp_output_t *   zoutput;
     pgp_output_t *   output;
     pgp_memory_t *   litmem;
     pgp_memory_t *   zmem;
     pgp_memory_t *   localmem;
-    unsigned         ret = 1;
+    unsigned         ret = 1;    
```

Comment:
Yup, seems you already fixed it in some other commit. But I've noticed some additional spaces where added here and there (like signature.c:954), so maybe try running clang-format again before commiting